js/liveblog-admin.js

```diff
@@ -9,17 +9,42 @@ jQuery( function( $ ) {
 		};
 	$meta_box.on( 'click', 'button', function( e ) {
 		e.preventDefault();
-		var url = ajaxurl + '?action=set_liveblog_state_for_post&post_id=' + encodeURIComponent( post_id ) + '&state=' + encodeURIComponent( $( this ).val() ) + '&' + liveblog_admin_settings.nonce_key + '=' + liveblog_admin_settings.nonce;
-		url += '&' + $('input, textarea, select', $meta_box).serialize();
-		$( '.inside', $meta_box ).load( url, function( response, status, xhr ) {
-			if ( status === 'success') {
-				$( 'p.success', $meta_box ).show(0).delay( 1000 ).hide(0);
-				return;
-			}
-			if (xhr.status && xhr.status > 200) {
-				show_error( xhr.statusText, xhr.status );
-			} else {
-				show_error( status );
+		var data = {};
+
+		if (liveblog_admin_settings.use_rest_api == 1) {
+			var url = liveblog_admin_settings.endpoint_url;
+			data['state']                           = encodeURIComponent( $( this ).val() );
+			data['template_name']                   = encodeURIComponent( $( '#liveblog-key-template-name' ).val() );
+			data['template_format']                 = encodeURIComponent( $( '#liveblog-key-template-format' ).val() );
+			data['limit']                           = encodeURIComponent( $( '#liveblog-key-limit' ).val() );
+			data[liveblog_admin_settings.nonce_key] = liveblog_admin_settings.nonce;
+			var method = 'POST';
```

Comment:
Will this mean all requests from the frontend are `POST` requests? As `POST`s will always be uncached, they'll create unnecessary load if that's the default.

Reply:
Ignore this :)

Reply:
👍